components/menu/menu.js

```diff
@@ -65,12 +65,12 @@ class Menu extends ThemeMixin(HierarchicalViewMixin(LitElement)) {
 			}
 
 			:host([theme="dark"]) {
-				--d2l-menu-background-color: #333536; /* ferrite @ 70% */
+				--d2l-menu-background-color: #333536; /* tungsten @ 70% */
 				--d2l-menu-background-color-hover: #123559; /* celestine-1 @ 50% */
-				--d2l-menu-border-color: var(--d2l-color-ferrite);
+				--d2l-menu-border-color: var(--d2l-color-tungsten);
 				--d2l-menu-foreground-color: var(--d2l-color-sylvite);
 				--d2l-menu-foreground-color-hover: #ffffff;
-				--d2l-menu-separator-color: var(--d2l-color-tungsten);
+				--d2l-menu-separator-color: var(--d2l-color-galena);
```

Comment:
Changed this one lighter to Galena so we get a different colour for the separator.